health-services/libraries/health-services-models/src/main/java/org/egov/common/models/project/useraction/UserActionSearch.java

```diff
@@ -0,0 +1,49 @@
+package org.egov.common.models.project.useraction;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+import org.egov.common.models.core.EgovOfflineSearchModel;
+import org.springframework.validation.annotation.Validated;
+
+/**
+ * The UserActionSearch class is used for searching user actions based on various criteria.
+ * It extends the EgovOfflineSearchModel to inherit common search properties.
+ */
+@Validated
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@SuperBuilder
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class UserActionSearch extends EgovOfflineSearchModel {
+
+    /**
+     * A list of project IDs to filter the user actions.
+     */
+    @JsonProperty("projectId")
```

Comment:
can only search for one project at a time. no need for an array.

Reply:
done